app/build.gradle

```diff
@@ -27,6 +27,8 @@ android {
         ndk {
             moduleName "xtrs"
         }
+
+        vectorDrawables.generatedDensities = ['mdpi','hdpi','xhdpi']
```

Comment:
On second viewing, it appears that its not needed because the support library version is greater that 23.2.  I'll remove it.

> If you want to use vector drawables only, you can use Android Support Library 23.2 or higher. This technique requires a change to your build.gradle file before you run Vector Asset Studio